pkg/skaffold/render/transform/transform.go

```diff
@@ -26,10 +26,32 @@ import (
 )
 
 var (
-	allowListedTransformer = []string{"set-label"}
-	transformerAllowlist   = map[string]kptfile.Function{
-		"set-label": {
-			Image:     "gcr.io/kpt-functions/set-namespace",
+	allowListedTransformer = []string{
+		"apply-setters",
+		"create-setters",
+		"set-annotations",
+		"set-labels",
+		"set-namespace",
+	}
+	transformerAllowlist = map[string]kptfile.Function{
```

Comment:
compute the key-set from `transformerAllowlist` where required and remove `allowListedTransformer` since they're the same.

Reply:
I'm fine with computing the names. But it is less efficient than just listing the accepted keys here.

Reply:
at this map size, efficiency shouldn't be an issue. I'm more concerned about not duplicating the same lookup data, since they need to be manually kept in sync.